corehq/couchapps/cases_by_server_date/views/by_server_modified_on/map.js

```diff
@@ -0,0 +1,5 @@
+function (doc) {
+    if (doc.doc_type === "CommCareCase") {
+        emit([doc.domain, doc._id], doc.server_modified_on);
```

Comment:
The view name doesn't seem to agree with the code since you can't query based on the modified on date. Maybe this should just be called `case_server_modified_on`